draft-packages/avatar/docs/Avatar.stories.tsx

```diff
@@ -40,121 +36,261 @@ DefaultStory.args = {
+          <Avatar
+            fullName="李存信"
+            disableInitials={false}
+            isCurrentUser
+            size="small"
+          />
+          <Avatar
+            fullName="Spicy Jalapeno Bacon Ipsum Dolor Amet Aute Elit Chicken Mollit"
+            disableInitials={false}
+            isCurrentUser
+            size="small"
+          />
+          <Avatar
+            fullName="Hooli"
+            avatarSrc="https://d1e7r7b0lb8p4d.cloudfront.net/third-party-logos/hooli-logo.svg"
+            isCompany
+            size="small"
+          />
+        </StoryWrapper.Row>
+      </StoryWrapper>
+    </div>
+  </>
 )
 
-DesignSheetDefault.storyName = "Design Sheet (default)"
-
-export const DesignSheetReversed = () => (
-  <div
-    style={{
-      display: "grid",
-      gridTemplateColumns: "repeat(auto-fit, minmax(150px, max-content))",
-      gap: "2.5rem",
-    }}
-  >
-    {avatarStoryData["users"].map(story => (
-      <div>
-        <Heading color="white" tag="h2" variant="heading-2">
-          {story.title}
-        </Heading>
-        <br />
-        {(story.stories as AvatarProps[]).map((storyData: AvatarProps) => {
-          const avatarProps = storyData as AvatarProps
-          return (
-            <>
-              <Avatar {...avatarProps} />
-              <br />
-            </>
-          )
-        })}
-      </div>
-    ))}
-  </div>
-)
-
-DesignSheetReversed.storyName = "Design Sheet (reversed)"
-DesignSheetReversed.parameters = {
-  backgrounds: { default: "Purple 700" },
-}
-
-export const InitialsLong = () => (
-  <div
-    style={{
-      display: "grid",
-      gridTemplateColumns: "repeat(auto-fit, minmax(150px, max-content))",
-      gap: "2.5rem",
-    }}
-  >
-    {avatarStoryData["initials-long"].map(story => (
-      <div>
-        <Heading tag="h2" variant="heading-2">
-          {story.title}
-        </Heading>
-        <br />
-        {(story.stories as AvatarProps[]).map((storyData: AvatarProps) => {
-          const avatarProps = storyData as AvatarProps
-          return (
-            <>
-              <Avatar {...avatarProps} />
-              <br />
-            </>
-          )
-        })}
-      </div>
-    ))}
-  </div>
-)
-
-export const InitialsUnicode = () => (
-  <div
-    style={{
-      display: "grid",
-      gridTemplateColumns: "repeat(auto-fit, minmax(150px, max-content))",
-      gap: "2.5rem",
-    }}
-  >
-    {avatarStoryData["initials-unicode"].map(story => (
-      <div>
-        <Heading tag="h2" variant="heading-2">
-          {story.title}
-        </Heading>
-        <br />
-        {(story.stories as AvatarProps[]).map((storyData: AvatarProps) => {
-          const avatarProps = storyData as AvatarProps
-          return (
-            <>
-              <Avatar {...avatarProps} />
-              <br />
-            </>
-          )
-        })}
-      </div>
-    ))}
-  </div>
-)
+StickerSheetDefault.storyName = "Sticker Sheet (Default)"
```

Comment:
We should probably still have a Reversed sticker sheet so we can visual test it both manually and in Chromatic.

One thing I looked into with the code sample and reusing story content is that we can't just make a content component then put it in, you actually need to create a Template and use the `Template.bind` pattern. Feel free to reach out if you need more clarification :)

Reply:
There are no isReversed or reversed prop/styles for the component so I assumed it was not necessary to create a ReversedSheet/ test within chromatic?

Reply:
Hmm...that's true too. It certainly makes it seem redundant. I just checked Figma and we don't have an example there either. Maybe it's okay to not have it? I might pose the question in Slack :)